docs/Build-for-RPi2.md

```diff
@@ -3,10 +3,11 @@
 ### Prerequisite
 #### Linux
 
-Install arm linux cross compiler.
+Install arm linux cross compiler and i2c library.
 
 ``` bash
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf i2c-tools libi2c-dev
+sudo mv /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h.kernel
```

Comment:
@akosthekiss  I found another I2C library under MIT license. (https://github.com/roadnarrows-robotics/i2c)

I2c codes in Intel's mraa library(https://github.com/intel-iot-devkit/mraa/blob/master/src/i2c/i2c.c) also use it.

How do you think modify it for us? If we have our own source codes for I2C, it is not needed to install libi2c-dev package.